src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/ApprenticeController.cs

```diff
@@ -28,5 +65,22 @@ public async Task<IActionResult> Details(DetailsRequest request)
             var viewModel = await _modelMapper.Map<DetailsViewModel>(request);
             return View(viewModel);
         }
+
+
+
+        [HttpGet]
+        [Route("download", Name = RouteNames.DownloadApprentices)]
+        public async Task<IActionResult> Download(long providerId, ApprenticesFilterModel filterModel)
```

Comment:
Same comment applies here as to the Index action - there should be a single parameter representing the request made (`DownloadRequest`) rather than the request being instantiated within the action.

Reply:
Changed so there is now `DownloadRequest`